### PR TITLE
DEVDOCS-5531: [revise]Update consignment endpoints

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5531]
{Ticket number or summary of work}

## What changed?
* Removed the array in the Update Checkout consignment and changed it to an object.
* Changed `address` to `shipping address` in the Add Consignment to Checkout
* Updated notes in the Update Checkout consignment endpoint about updating shipping options and addresses separately.

## Anything else?
Add related PRs, salient notes, ticket numbers, etc.

ping {names}

[DEVDOCS-5531]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5531?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/checkouts.v3.yml [hunk 1]
```diff
@@ -8619,7 +8622,7 @@ components:
       items:
         type: object
         properties:
-          address:
+          shipping_address:
```
Comment (bc-shawnwang) [inline reference/checkouts.v3.yml:8625]: I remember we allow both `address` and `shipping_address` (we defined them as alias). am I right? @valentindellangela

File: reference/checkouts.v3.yml [hunk 2]
```diff
@@ -8703,10 +8706,8 @@ components:
       x-internal: false
     UpdateConsignmentRequest:
       title: Update Consignment Request
-      type: array
-      items:
-        type: object
-        properties:
+      type: object
```
Comment (bc-shawnwang) [inline reference/checkouts.v3.yml:8709]: thanks for the change, this looks correct for me

File: reference/checkouts.v3.yml (hunk 1 above)
Comment (bc-shawnwang) [reply reference/checkouts.v3.yml:8625]: image

`shipping_address` (deprecated) or `address`

This is from storefront doc description. I checked the s2s part, it should be the same. and we encourage to use address

**Review by bc-tgomez — APPROVED** [review]
Looks good!

**Pull request merged**